<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Settings</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>271</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Saturation</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Contrast</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Brightness</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSlider" name="sliderContrast">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSlider" name="sliderSaturation">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSlider" name="sliderBrightness">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>550</width>
     <height>272</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="2">
     <widget class="QPlainTextEdit" name="headUpperHSV">
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPlainTextEdit" name="lekLowerHSV">
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPlainTextEdit" name="neckLowerHSV">
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="headLowerHSV">
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPlainTextEdit" name="rhfLowerHSV">
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPlainTextEdit" name="lekUpperHSV">
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Left Elbow&amp;Knee</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Right Hand&amp;Foot</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Hips </string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Left Hand&amp;Foot</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Neck </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Head </string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Right Elbow&amp;Knee</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>UPPER HSV</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Lower HSV</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPlainTextEdit" name="hipsUpperHSV">
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPlainTextEdit" name="neckUpperHSV">
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPlainTextEdit" name="hipsLowerHSV">
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPlainTextEdit" name="lhfLowerHSV">
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPlainTextEdit" name="rekLowerHSV">
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPlainTextEdit" name="lhfUpperHSV">
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPlainTextEdit" name="rekUpperHSV">
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>28</height>
       </size>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPlainTextEdit" name="rhfUpperHSV">
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Adjust Input Video</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>351</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Adjust Color HSV Thresholds (0-179, 0-255, 0-255)</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>490</y>
     <width>166</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>330</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Character</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="characterSimple">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Simple</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="characterMickey">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>390</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Mickey Mouse</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="rotateVideo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotate video 90 degrees</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="characterMorty">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>420</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Morty</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
